docs/howto_guides/sample_data.rst

```diff
@@ -0,0 +1,43 @@
+.. _dkist:howto-guide:sample-data:
+
+Downloading the Sample Data with Globus
+=======================================
+
+The Python tools provide a few different partial datasets as sample data, these are used in documentation and examples.
+This how to guide will show you how to download the complete data for these datasets, or let you change the dataset for a different one.
+
+
+Searching for and downloading the ASDF file
+-------------------------------------------
+
+Using the `BKPLX <https://dkist.data.nso.edu/datasetview/BKPLX>`__ dataset from the VISP we download the ASDF file and create a `dkist.Dataset` object.
```

Comment:
Looking to the future that [this PR is creating](https://bitbucket.org/dkistdc/dkist-inventory/pull-requests/60) we should probably replace references to an "ASDF file" with something like "metadata file".